modules/rostests/apitests/shell32/ShellExecuteEx.cpp

```diff
@@ -368,26 +368,40 @@ static void test_properties()
 
 static void test_sei_lpIDList()
 {
-    if (IsWindowsVistaOrGreater())
+    // Note: SEE_MASK_FLAG_NO_UI prevents the test from blocking with a MessageBox
```

Comment:
Please move this comment down to line 386 where this flag is being specified.

Reply:
That flag is used twice in this function which is why I did it up here. The reason I added the comment in the first place is that in my VM when the PIDL handling is broken, the whole test stops waiting for a MessageBox inside ShellExecuteEx.